subprojects/docs/src/docs/userguide/img/java-library-ignore-deprecated.dot

```diff
@@ -1,38 +1,39 @@
 digraph configurations {
-   rankdir = BT;
+    rankdir = BT;
    graph [ dpi = 100, fontname="Meera"];
    node [fontname = "Meera"];
    edge [fontname = "Meera"];
 
-   "api" [shape="box", label=<<B>api</B>>, color="chartreuse3", bgcolor="chartreuse3", style="filled"]
-   "apiElements" [shape="box", label=<<B>apiElements</B>(C)<BR/><FONT POINT-SIZE="8">usage:for compile</FONT>>, color="pink", bgcolor="pink", style="filled"]
+   "apiElements" [shape="box", label=<<B>apiElements</B>(C)<BR/><BR/><FONT POINT-SIZE="8">org.gradle.api.attributes.Usage:for compile</FONT>>, color="pink", bgcolor="pink", style="filled"]
    "apiElements" -> "api"
 
-   "compileClasspath" [shape="box", label=<<B>compileClasspath</B>(R)<BR/><FONT POINT-SIZE="8">usage:for compile</FONT>>, color="lightblue", bgcolor="lightblue", style="filled"]
+   "api" [shape="box", label=<<B>api</B><BR/><BR/>>, color="chartreuse3", bgcolor="chartreuse3", style="filled"]
+
+   "compileClasspath" [shape="box", label=<<B>compileClasspath</B>(R)<BR/><BR/><FONT POINT-SIZE="8">org.gradle.api.attributes.Usage:for compile</FONT>>, color="lightblue", bgcolor="lightblue", style="filled"]
    "compileClasspath" -> "compileOnly"
+   "compileOnly" [shape="box", label=<<B>compileOnly</B>(C, R)<BR/><BR/>>, ]
    "compileClasspath" -> "implementation"
-   "compileOnly" [shape="box", label=<<B>compileOnly</B>(C, R)>, ]
-   "implementation" [shape="box", label=<<B>implementation</B>>, color="chartreuse3", bgcolor="chartreuse3", style="filled"]
+   "implementation" [shape="box", label=<<B>implementation</B><BR/><BR/>>, color="chartreuse3", bgcolor="chartreuse3", style="filled"]
    "implementation" -> "api"
 
-   "runtimeOnly" [shape="box", label=<<B>runtimeOnly</B>>, color="chartreuse3", bgcolor="chartreuse3", style="filled"]
-   "runtimeClasspath" [shape="box", label=<<B>runtimeClasspath</B>(R)<BR/><FONT POINT-SIZE="8">usage:for runtime</FONT>>, color="lightblue", bgcolor="lightblue", style="filled"]
+   "runtimeClasspath" [shape="box", label=<<B>runtimeClasspath</B>(R)<BR/><BR/><FONT POINT-SIZE="8">org.gradle.api.attributes.Usage:for runtime</FONT>>, color="lightblue", bgcolor="lightblue", style="filled"]
    "runtimeClasspath" -> "runtimeOnly"
+   "runtimeOnly" [shape="box", label=<<B>runtimeOnly</B><BR/><BR/>>, color="chartreuse3", bgcolor="chartreuse3", style="filled"]
    "runtimeClasspath" -> "implementation"
-   "runtimeElements" [shape="box", label=<<B>runtimeElements</B>(C)<BR/><FONT POINT-SIZE="8">usage:for runtime</FONT>>, color="pink", bgcolor="pink", style="filled"]
+   "runtimeElements" [shape="box", label=<<B>runtimeElements</B>(C)<BR/><BR/><FONT POINT-SIZE="8">org.gradle.api.attributes.Usage:for runtime</FONT>>, color="pink", bgcolor="pink", style="filled"]
```

Comment:
It was there before, but now that you say it, it might just be confusing without prior explanation. Also, it was accidentally replaced with qualified names. I'll probably get rid of it.